### ISSUE TITLE
Deployment fails

### ISSUE THREAD

**Issue by olimart** [fastmonkeys/sentry-on-heroku#18]
Hi,
thanks for putting this together. I'm running into an issue when using heroku button to deploy. Here is the trace. Any idea?

```
Traceback (most recent call last):
  File "/app/.heroku/python/lib/python2.7/site-packages/logan/runner.py", line 116, in settings_callback
    'settings': settings,
  File "/app/.heroku/python/lib/python2.7/site-packages/sentry/utils/runner.py", line 317, in initialize_app
    validate_backends()
  File "/app/.heroku/python/lib/python2.7/site-packages/sentry/utils/runner.py", line 328, in validate_backends
    app.tsdb.validate()
  File "/app/.heroku/python/lib/python2.7/site-packages/sentry/tsdb/redis.py", line 74, in validate
    raise InvalidConfiguration(unicode(e))
InvalidConfiguration: max number of clients reached
```

**Comment by jpvanhal** [comment]
I think you need a larger plan for Heroku Redis. The free plan has connection limit of 20.

**Comment by olimart** [comment]
Thanks @jpvanhal Is there a way to decrease the limit in codebase?

**Comment by jpvanhal** [comment]
Fixed in 48f30774ce917c980004bd721e1dfe931e06fac3. I changed the Heroku Redis plan from Hobby Dev to Premium 0

**Comment by jpvanhal** [comment]
> Thanks @jpvanhal Is there a way to decrease the limit in codebase?

I don't think there's any way of limiting or decreasing how many connections Sentry makes to Redis.
